src/components/CommentList.jsx

```diff
@@ -13,7 +13,7 @@ const CommentList = () => {
       setComments(comments);
       setIsLoading(false);
     });
-  }, [comments]);
+  }, [updateComments]);
```

Comment:
This is far from ideal, I made 75 requests to your database just writing out a comment, this re renders your comments every time you type a character...